src/main/java/io/github/thebusybiscuit/cscorelib2/protection/ProtectionManager.java

```diff
@@ -7,6 +7,7 @@
 import java.util.logging.LogRecord;
 import java.util.logging.Logger;
 
+import io.github.thebusybiscuit.cscorelib2.protection.modules.*;
```

Comment:
Avoid the usage of wildcard imports.

Reply:
To be honest, I don't see any reason not to use wildcard operator in that case. ProtectionManager imports all protection modules. When it would be about importing only some of the classes from the modules package, I would agree with you.

Reply:
Wildcard imports are just generally something we want to avoid here.
Yes, in this instance it won't do any harm but wildcards are generally to be avoided to prevent namespace pollution.

You can see it as a "style preference" in this case. It's not breaking but following it would be appreciated.

Reply:
It only makes the code shorter with no drawbacks (in this case of course). But fine, changed.